readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://natalia-maryshko.github.io/layout_stop-watch/)
```

Comment:
<img width="885" alt="image" src="https://user-images.githubusercontent.com/38065883/172705043-837f1b8f-ef1e-425c-860f-551fc68b3dd4.png">
Would be nice if clock wouldn't skew if window gets smaller